frontend/catalyst/pennylane_extensions.py

```diff
@@ -74,7 +74,12 @@ def __call__(self, *args, **kwargs):
                     f"The {self.device.short_name} device is not "
                     "supported for compilation at the moment."
                 )
-            device = QJITDevice(self.device.shots, self.device.wires, self.device.short_name)
+            device_name = (
+                self.device.short_name
+                if self.device.short_name != "braket.simulator"
+                else "arn:aws:braket:::device/quantum-simulator/amazon/sv1"
```

Comment:
@maliasadi should we use the same naming convention that the [Braket PL plugin](https://amazon-braket-pennylane-plugin-python.readthedocs.io/en/latest/devices/braket_remote.html) already uses. That is,

```python
qml.device("braket.aws.qubit", device_arn="arn:aws:braket:::device/quantum-simulator/amazon/sv1", s3_destination_folder=s3, wires=2)
```

Reply:
This way the user is able to pass in the device ARN separately

Reply:
Sure, we can. I'll update this.

Reply:
@josh146 I also should add that the frontend seems fragile at this point and this's mostly because of the fact that we were discussing the param-circuits support in the past few days and trying to keep things in the frontend simplified to not blocking anything for the other work... Now, we have a clearer path towards that support.

Reply:
Amazon resource name :)

Reply:
Addressed in PR #154.